generators/typescript/sdk/generator/src/SdkGenerator.ts

```diff
@@ -806,7 +806,8 @@ export class SdkGenerator {
                             },
                             snippet: FernGeneratorExec.EndpointSnippet.typescript({
                                 client: snippet
-                            })
+                            }),
+                            exampleIdentifier: example.name?.originalName
```

Comment:
On line 784 we do `const example = endpoint.examples[0]`, can we turn that into a for loop now?

Reply:
missed that, updated